tests/instrumentation-framework/src/main/java/org/hawkular/apm/tests/dockerized/TestScenarioRunner.java

```diff
@@ -84,25 +71,25 @@ public TestScenarioRunner(String apmVersion, int apmServerPort) throws DockerCer
     public int run(TestScenario testScenario) {
         if (testScenario.getEnvironment().getDockerCompose() != null &&
                 testScenario.getEnvironment().getImage() != null) {
-            throw new IllegalArgumentException("Ambiguous environment: defined docker image and docker-compose");
+            throw new IllegalArgumentException("Ambiguous environment: defined docker image and docker-compose" +
+                    ", but we expect only one of them to be defined!");
         }
 
-        System.out.println("Starting test scenario: " + testScenario);
+        log.info(String.format("Starting test scenario: %s", testScenario));
```

Comment:
It is fine for a test, but for production code, it would be best to not format the message and then call the log, because the message will always be formatted, even if the `info` level is not active. For production code, the best would be: `log.log(Level.INFO, message, testScenario)`. There's a similar feature on JBoss Logging. But again: no need to change this specific code (unless you want to, of course).